### ISSUE TITLE
web-wallet: No auto-sync after moonlight transfer

### ISSUE THREAD

**Issue by lufa23** [dusk-network/rusk#3002]
From Guestno1:

**Description**: 
While having wallet open i request faucet transfer. The transfer proceeds successfully. Transaction is shown in the explorer.
After 10 minutes of looking at the wallet directly (no alt tabing no nothing) the balance isn't updated. If i open wallet in another browser the funds are there. Navigating to different web wallet parts doesn't help. Only doing transaction or swap triggers sync and the funds are shown.

**Steps to reproduce**
Recover web wallet and finish syncing.
Request funds from faucet to that web wallet (i assume its equal to get a moonlight transfer to Your address but i didn't test)
Confirm transaction was successful.
Wait for 5 minutes.

**Expected behavior:**
Wallet auto-syncs maximum 5 minutes (or whatever interval is currently set to) after transaction is complete.

**Platform**
Win-11
Mozilla 132.0.2

**Comment by ascartabelli** [comment]
> Expected behavior:
Wallet auto-syncs maximum 5 minutes (or whatever interval is currently set to) after transaction is complete.

Actually that's not the expected behaviour, otherwise we wouldn't have to wait for five minutes and we could instantly update the wallet.
The auto-sync is a temporary measure to mitigate the fact that we are currently unable to see a transaction made to us.

The expected behaviour is that the web-wallet starts a new sync five minutes after the last one.

Currently the sync happens at login, before executing an operation and when the transaction that is the result of our operation is removed from the mempool.

**Comment by ascartabelli** [comment]
@lufa23 while waiting for further info on this, wanted to let you know that we found, and fixed, a bug that could cause the issue reported here: https://github.com/dusk-network/rusk/issues/3042

**Comment by lufa23** [comment]
yes, https://github.com/dusk-network/rusk/issues/3042 should most probably fix this. Once it's released the user will test again and confirm if the issue is solved
